lib/collection/src/collection_manager/optimizers/indexing_optimizer.rs

```diff
@@ -236,13 +236,15 @@ impl SegmentOptimizer for IndexingOptimizer {
     }
 
     fn get_telemetry_data(&self) -> OptimizerTelemetry {
-        OptimizerTelemetry::Indexing {
-            optimizations: self.get_telemetry_counter().lock().get_statistics(),
+        OptimizerTelemetry {
+            indexing: self.get_telemetry_counter().lock().get_statistics(),
+            merge: Default::default(),
```

Comment:
are we actually tracking `merge` and `vacuum`?

Reply:
We tracking all optimizers type. IMHO we can remove it from telemetry only when we remove such entities from code